core/src/main/scala/akka/kafka/scaladsl/Consumer.scala

```diff
@@ -111,18 +111,27 @@ object Consumer {
    * Materialized value of the consumer `Source`.
    */
   trait Control {
+
+    /**
+     * Stop producing messages from the `Source`. This does not stop underlying kafka consumer
+     * and does not unsubscribe from any topics/partitions.
+     *
+     * Call [[#shutdown]] to close consumer
+     */
+    def stop(): Unit
```

Comment:
If it complete when stage is shutdown then it never completes until someone call `shutdown`. This is look confusing for me. 

Typical case for me is:
- call `stop`
- wait for correct stream completion
- call shutdown at stream completion or after some timeout for stream completion.